app/models/event.rb

```diff
@@ -14,4 +14,8 @@ class Event < ApplicationRecord
   scope :requiring_notification, -> do
     where('date = ?', Date.current - NOTIFICATION_PERIOD)
   end
+
+  def duration
+    presentations.map(&:duration).sum
```

Comment:
No need to change this, just pointing out that this could also be performed at the db level with `#pluck` There are tradeoffs to both perfoming the calculation on the ruby side and doing on the db side. 

Theres no need to change this. Just wanted to point out a fun fact